packages/calcite-components/src/utils/locale.ts

```diff
@@ -212,10 +212,12 @@ export interface LocalizedComponent {
    *
    * Components should watch this prop to ensure messages are updated.
    *
+   * ```tsx
```

Comment:
Having `@Watch` in JSDoc without it being escaped in backticks was being parsed by TypeScript as a JSDoc tag, making TypeScript insert extra whitespace around it.
To have things parse correctly, it is advised to wrap all code blocks in JSDoc comments in backticks.

Reply:
Ah, I didn't realize that.
Looks like there is no legal way to write decorator code examples in JSDoc for now. There is an issue for TypeScript 5.7 though: https://github.com/microsoft/TypeScript/issues/47679
I will revert this change then, and add a post-processing step to the codemod to remove the extra whitespace added by incorrect parsing.